docs/blog/newsletter-july-2020.rst

```diff
@@ -0,0 +1,123 @@
+
+With that - what's the WTD community been talking about on Slack?
+
+----------------------------------------
+A magnificent array of management topics
+----------------------------------------
+
+This month treated us to several broad and deep discussions related to management. First up, **interest in managing vs. writing**: what should you do if you're not sure whether a tech writer on your team is interested in management duties?
+
+- Just ask! It's important to know up front if the answer is "definitely," "not at all," or something in between, so you can figure out if their level of interest matches your expectations.
+- If they do seem interested, be clear about the expected duties, and what control they'd have. Would they control hiring and performance reviews? Or would it be more around mentoring and training?
+- Consider options for testing the waters without committing to full-time management. Team or project lead can be a great starting point.
+- If they aren't interested, don't force it. If someone is happier as a hands-on contributor, they may be miserable in management.
+
+What if you're on the other side, **considering a move into management** - how do you decide?
+
+- Pros: you can achieve bigger projects as a manager than by yourself; you can help other technical writers grow, which can be extremely rewarding; if you enjoy facilitating and attending meetings, you'll be in your element.
+- Cons: many writers just want to write, and there can be less time for that; at some point, you may have to shoulder the burden of firing; plus these `17 Reasons Not to Be a Manager <https://charity.wtf/2019/09/08/reasons-not-to-be-a-manager>`_.
+- Check if you're doing management work already without realising it. Perhaps you take a special joy in managing and planning projects, including making sure your teammates feel good about projects and their work. Or, maybe you're helping with team morale, eliminating barriers to people getting work done, or supporting your peers' career growth.
+- Taking on management responsibilities without being a lead/senior can change how you're perceived, so it's worth being thoughtful about it. `This piece by Tanja Reilly <https://noidea.dog/glue>`_ describes how such so-called "glue work" can affect your career.
+
+-------------------------------
+Gathering documentation feedback
+-------------------------------
+
+To improve documentation, we need to know where it's lacking, and we can get this feedback from a variety of sources, including user interviews and internal surveys. Here's some advice from our community on how to go about it!
```

Comment:
This first sentence feels a little choppy. I don't have a great suggestion for improvement though.

Reply:
Perhaps:
```suggestion
To improve documentation, we need to know where it's lacking. We can get this feedback from a variety of sources, including user interviews and internal surveys. Here's some advice from our community on how to go about it!
```

Reply:
Solid choice.